util/BUILD

```diff
@@ -78,3 +78,11 @@ py_binary(
         requirement("pluralizer"),
     ],
 )
+
+py_test(
```

Comment:
Do we run this in CI?

Reply:
[Yes](https://dev.azure.com/lowrisc/opentitan/_build/results?buildId=105310&view=logs&j=5df5adf8-3577-5927-f0b2-31afed2697ee&t=1253ba27-4e8c-57af-d0ba-888237644875&l=3479)

The Earl Gray SW Build and Test job should pick up any stray tests in the repo that aren't part of the "device" testing (e.g. functests and e2e tests)